specs/trip_dispatch_spec.rb

```diff
@@ -122,4 +121,47 @@ def build_test_dispatcher
       end
     end
   end
+
+  describe "request_trip method" do
+    before do
+      @dispatcher = build_test_dispatcher
+    end
+    it "creates a new instance of a trip" do
+      starting_length = @dispatcher.trips.length
+      test_passenger = @dispatcher.passengers[3]
+      test_passenger_id = test_passenger.id
+      @dispatcher.request_trip(test_passenger_id)
+      ending_length = @dispatcher.trips.length
+      expect(ending_length).must_equal starting_length + 1
+    end
+
+    it "sets the driver's status to :UNAVAILABLE and adds to driver class" do
+      trip = @dispatcher.request_trip(5)
+      expect(trip.driver.status).must_equal :UNAVAILABLE
```

Comment:
You should also test that the driver *was* available before the trip was requested.